eng/tools/compare-packages.ps1

```diff
@@ -0,0 +1,60 @@
+param (
+  $pathToMasterPkg,
+  $pathToCurrentPkg,
+  $dailyDevBuildNo,
+  $pathForDiffFile
+)
+
+function ExtractTGZPackages($pathToPkg){
```

Comment:
nit: Generally people put a space before the `{`. This applies to most of this file.

Reply:
done